### ISSUE TITLE
Flag ignore don't work

### ISSUE THREAD

**Issue by stamm** [rubysec/bundler-audit#95]
Run bundle audit with ignore particular version of CVE, but it find this vulnerability
```bash
✗ bundle-audit check --ignore "CVE-2014-4920"
Insecure Source URI found: git://github.com/airblade/paper_trail.git
....
Name: twitter-bootstrap-rails
Version: 2.2.8
Advisory: CVE-2014-4920
Criticality: Unknown
URL: http://blog.nvisium.com/2014/03/reflected-xss-vulnerability-in-twitter.html
Title: Reflective XSS Vulnerability in twitter-bootstrap-rails
Solution: upgrade to >= 3.2.0

Vulnerabilities found!
```

**Comment by latortuga** [comment]
This looks like it was probably fixed by 4e32fca89d75f0e249671431ff38aadc02bfb28b.

**Comment by stamm** [comment]
Yeap, working on master
